egs/wsj/s5/utils/data/get_subsegmented_feats.sh

```diff
@@ -0,0 +1,52 @@
+#! /bin/bash
+
+# Copyright 2016  Johns Hopkins University (Author: Dan Povey)
+#           2016  Vimal Manohar
+# Apache 2.0.
+
+if [ $# -ne 4 ]; then
+  echo "This scripts gets subsegmented_feats (by adding ranges to data/feats.scp) "
```

Comment:
I don't believe you addressed this comment, and I downloaded your branch manually so I think it's not just github.